cypress/e2e/registration-form.cy.ts

```diff
@@ -0,0 +1,34 @@
+import {StudentRegistrationForm} from "../page/index";
+import {PersonalInformation} from "../interfaces/PersonalInformation";
+
+describe("Personal form page", () => {
+  let personalFormPage: StudentRegistrationForm;
+  let personalInfo: PersonalInformation;
+
+  before(() => {
+    personalFormPage = new StudentRegistrationForm();
+    personalInfo = {
+      name: "Holmes",
+      lastName: "Salazar",
+      email: "test@email.com",
+      gender: "Male",
+      dateOfBirth: "27 Jul 2016",
+      mobileNumber: "3656589156",
+      hobbies: ["1", "2", "3"],
```

Comment:
seeing hobbies as integer is not readable, use the description

Reply:
resolved